File: server/src/main/java/com/ps/server/dto/MoveUpdateDTO.java
package com.ps.server.dto;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.validation.constraints.NotNull;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
public class MoveUpdateDTO {

    private Long updateId = 0L;

    @NotNull
    private MoveResponseDTO moveDTO;

    public Long getUpdateId() {
        return updateId;
    }

    public MoveResponseDTO getMoveDTO() {
        return moveDTO;
    }

    public void setUpdateId(Long updateId) {
        this.updateId = updateId;
    }

    public void setMoveDTO(MoveResponseDTO moveDTO) {
        this.moveDTO = moveDTO;
    }
}

File: client/src/app/DTO/moveUpdateDTO.ts
import {MoveResponseDTO} from "./moveResponseDTO";

/** @ignore*/
export class MoveUpdateDTO {
    /** */
    public updateId: number;
    /** */
    public moveDTO: MoveResponseDTO;

    /** @ignore*/
    constructor(updateId: number, moveDTO: MoveResponseDTO) {
        this.updateId = updateId;
        this.moveDTO = moveDTO;
    }
}